GitSearch/Networking/HTTP/HTTPNetworkRequest.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  HTTPNetworkRequest.swift
+//  GitSearch
+//
+//  Created by Vldmr on 8/27/19.
+//  Copyright © 2019 mudritskiy. All rights reserved.
+//
+
+import Foundation
+
+public typealias HTTPParameters = [String: Any]?
+public typealias HTTPHeaders = [String: Any]?
+
+struct HTTPNetworkRequest {
+    
+    static var urlComponents: URLComponents = {
+        var url = URLComponents()
+        url.scheme = "https"
+        url.host = "api.github.com"
+        return url
+    }()
+    
+    static func configureHTTPRequest(from route: HTTPNetworkRoute, with parameters: HTTPParameters, includes headers: HTTPHeaders, contains body: Data?, and method: HTTPMethod) throws -> URLRequest {
+        
+        urlComponents.path = route.rawValue
+        guard let url = urlComponents.url else {
+            throw HTTPNetworkError.missingURL
+        }
+
+        var request = URLRequest(url: url, cachePolicy: .reloadIgnoringLocalCacheData, timeoutInterval: 10.0)
```

Comment:
Why to disable caching? And why so small timeout?

Reply:
need time to get it